packages/utilities/psammead-test-helpers/src/index.js

```diff
@@ -5,7 +5,7 @@ import deepClone from 'ramda/src/clone';
 export const shouldMatchSnapshot = (title, component) => {
   it(title, () => {
     const { container } = render(component);
-    expect(container.firstChild).toMatchSnapshot();
```

Comment:
What about something like

```js
const hasOneChild = container.children.length === 1
const removeWrappingDiv = container => container.firstChild

expect(hasOneChild ? removeWrappingDiv(container) : container).toMatchSnapshot();
```

Reply:
Thanks for the help @jroebu14 with this logic I don't have to go updating every snapshot in psammead 😅 

Also we tested it against this PR: https://github.com/bbc/simorgh/pull/3375 by copying the `esm` and `dist` of `psammead-test-helper` directly into the Simorgh node_modules and it passes and fixes the snapshot issue we were having. 👍 

Thanks again 💯